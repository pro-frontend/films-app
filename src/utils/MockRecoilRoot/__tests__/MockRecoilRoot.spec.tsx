import { mount, ReactWrapper } from "enzyme";
import toJson from "enzyme-to-json";
import React from "react";
import { MockRecoilRoot } from "../";

let component: ReactWrapper;

describe("<MockRecoilRoot />", () => {
  beforeEach(() => {
    component = mount(<MockRecoilRoot />);
  });

  it("should mount MockRecoilRoot", () => {
    expect(toJson(component)).toMatchSnapshot();
  });
});
