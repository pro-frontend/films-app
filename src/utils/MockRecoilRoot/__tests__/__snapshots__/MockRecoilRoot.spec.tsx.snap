// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MockRecoilRoot /> should mount MockRecoilRoot 1`] = `
<MockRecoilRoot>
  <RecoilRoot>
    <Batcher
      setNotifyBatcherOfChange={[Function]}
    />
  </RecoilRoot>
</MockRecoilRoot>
`;
